src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/netty/handlers/MainHandler.kt

```diff
@@ -45,11 +46,12 @@ class MainHandler(
     override fun decode(ctx: ChannelHandlerContext, buf: ByteBuf, out: MutableList<Any>) {
         logger.trace { "Attempting to decode data received from: ${ctx.channel().remoteAddress()} (data: ${hexDump(buf)})" }
 
+        val receiveTime = Instant.now()
```

Comment:
Please move this instruction to the first line of this method

Reply:
I don't think it will change anything. However, if you ask - sure